examples/cmd/vlan_detail.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/jedib0t/go-pretty/v6/table"
+
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+)
+
+func init() {
+	rootCmd.AddCommand(detailVlanCmd)
+}
+
+var detailVlanCmd = &cobra.Command{
+	Use:   "vlan-detail [vlanId]",
+	Short: "Vlan Details",
+	Long:  `Gets a lot of information about a VLAN`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("a VLAN ID is required")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return VlanDetailCommand(cmd, args)
+	},
+}
+
+func VlanDetailCommand(cmd *cobra.Command, args []string) error {
+	// resultLimit := 50
+	// resultOffset := 0

```

Comment:
Please remove these two comments lines `resultLimit` and `resultOffset` because it seems that they are not necessary.